test/run

```diff
@@ -16,7 +16,7 @@ source $(dirname $0)/common
 
 cd ${ROOT}
 
-extra_args="--ursula-user=${LOGIN_USER} --ursula-debug --provisioner=heat --heat-stack-name=${testenv_heat_stack_name}"
+extra_args="--ursula-user=${LOGIN_USER} --ursula-debug --ursula-sudo"
```

Comment:
interesting, u just posted comment and it triggered CI build. Might want to look into it.